__tests__/app.test.js

```diff
@@ -296,3 +296,44 @@ describe("POST: /api/articles/:article_id/comments", () => {
         .expect(404)
     });
 })
+
+describe("PATCH /api/articles/:article_id", () => {
+    test("200: Successful patch will return the article", () => {
```

Comment:
We should have a test that checks whether the request is also able to decrement the votes on the article